### PR TITLE
Demo feature integrate recipe chef

### PR DESCRIPTION
**Pull request by MinaKhamesi**
Chef Profile page now has a functional button to add recipes and recipe component has the functionality to invoke the modal in edit mode. for now you have to click on the picture of the recipe to invoke the modal

### CODE REVIEW

File: client/src/components/ChefProfile/ChefRecipes.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+  // const {
+  //   name,
+  //   pictureUrl,
+  //   price,
+  //   ingredients,
+  //   requiredStuffs,
+  //   portionDescription,
+  //   cuisineTags,
+  // } = props.recipe;
```
Comment (dproc96) [inline client/src/components/ChefProfile/ChefRecipes.js:63]: If this commented out code isn't needed, you can probably remove it and include it when it's needed

File: client/src/pages/ChefProfile.js [hunk 2]
```diff
@@ -1,16 +1,159 @@
...
+  //         "soy sauce",
+  //         "salt",
+  //         "sugar",
+  //       ],
+  //       requiredStuffs: ["pan", "oil", "kitchen table"],
+  //       portionDescription: "meal for 2",
+  //       cuisineTags: ["european"],
+  //     },
+  //   ],
+  // };
```
Comment (dproc96) [inline client/src/pages/ChefProfile.js]: Also for this commented out code, you can remove this if it's unused

**Review by dproc96 — COMMENTED** [review]
It looks like there are a number of warnings in the console for this, can you take a look at that? ~Additionally, I'm getting a 404 error and being redirected to "/profile" when I try to go to "/chefprofile". Any idea why?~

Took another look and figured that problem out, pending the warnings this is good to go

**Review by dproc96 — APPROVED** [review]

File: client/src/pages/ChefProfile.js (hunk 2 above)
Comment (MinaKhamesi) [reply client/src/pages/ChefProfile.js]: Thanks Dan, Sure I will delete them.

**Pull request merged**